<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HTJJPCU" failures="1" tests="7" name="Test" time="11.418" errors="0" timestamp="2024-10-27T13:19:53 IST">
  <testcase classname="api.tests.TestUser" name="testPostUser" time="4.126"/>
  <testcase classname="api.tests.TestUser" name="testGetUser" time="1.263"/>
  <testcase classname="api.tests.TestUser" name="testUpdateUser" time="1.194"/>
  <testcase classname="api.tests.TestUser" name="testGetUser2" time="1.725"/>
  <testcase classname="api.tests.TestUser" name="testDeleteUser" time="1.19"/>
  <testcase classname="api.tests.TestUser" name="testGetUser3" time="1.21">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.TestUser.testGetUser3(TestUser.java:92)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testGetUser3 -->
  <testcase classname="api.tests.TestUser" name="testGetUser4" time="0.545"/>
</testsuite> <!-- Test -->
